.circleci/smoke_test/docker/Dockerfile

```diff
@@ -25,12 +25,12 @@ RUN apt-get -qq update && apt-get -qq -y install curl bzip2 sox libsox-dev libso
 
 ENV PATH /opt/conda/bin:$PATH
 
-RUN conda create -y --name python3.5 python=3.5
 RUN conda create -y --name python3.6 python=3.6
 RUN conda create -y --name python3.7 python=3.7
+RUN conda create -y --name python3.8 python=3.8
```

Comment:
Why do we give up 3.5?

Reply:
PyTorch master branch no longer supports 3.5.